pyproject.toml

```diff
@@ -7,6 +7,8 @@ license = "MIT"
 
```

Comment:
You should add a name, authors, description, and other metadata for your project. Eventually you will need to publish this project on PyPI, and this metadata will be useful!